src/DiscSimulations.jl

```diff
@@ -7,10 +7,32 @@ using Printf
 include("solution.jl")
 
 include("burgers/utils.jl")
-include("burgers/spectral.jl")
-include("burgers/simple.jl")
-include("burgers/trixi.jl")
+#include("burgers/spectral.jl")
+#include("burgers/simple.jl")
+include("simpleGravitySim/simulation.jl")
 include("younsi-2012.jl")
+include("Euler_with_source.jl")
+include("burgers/trixi.jl")
+
+struct Parameters{}
+    N
+    xmin
+    xmax
+    ρ
+    source
+end
+
+abstract type SimulationType end
+main(p::Parameters, ::SimulationType) = error("Unknown simulation type.")
```

Comment:
Nice! :) Maybe also feedback what the time is, so a user knows what to correct, along the line of

```julia
main(p::Parameters, t::SimulationType) = error("Unknown simulation type: $(typeof(t))")
```